<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
	background-color: #3a3e47;
}

QLabel {
	color: orange;
	font-size: 15pt;
}

#longitudeEdit, #lattitudeEdit, #scaleSpinBox, #mapTypeComboBox, #objectAddressEditl, #postIndexComboBox, #objectAddressEdit, #fullAddressLabel
 {
	background-color: transparent;
	border: 2px solid white;
	border-radius: 10px;
	color: white;
	font-size: 12pt;
}

#showMapButton, #addPointButton, #resetButton {
	color: orange;
	background-color: transparent;
	border: 2px solid white;
	border-radius: 10px;
	font-size: 15pt;
}

#showMapButton:hover, #addPointButton:hover, #resetButton:hover {
	background-color: #0cf7dc;
}

#showMapButton:pressed, #addPointButton:pressed, #resetButton:pressed {
	background-color: pink;
}

#mapImage {
	border: 1px solid white;
	border-radius: 1px;
}

QRadioButton {
	background-color: transparent;
	color: white;
	font-size: 15pt;
}

#line {
	color: white;
}

#fullAddressLabel {
	font-size: 10pt;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="scaleSpinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>17</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="longitudeEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lattitudeEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="showMapButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Показать карту</string>
    </property>
   </widget>
   <widget class="QLabel" name="scaleLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Масштаб</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="longitudeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Долгота</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lattitudeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ширина</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mapImage">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>650</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="mapTypeComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="mapTypeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Тип карты</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="objectAddressLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес объекта</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="objectAddressEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>321</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="postIndexLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Почтовый индекс</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="postIndexOnRadioButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>590</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Да</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="postIndexOffRadioButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>620</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Нет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPointButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>680</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить метку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>680</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Сброс</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>460</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="full_address">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>500</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Полный адрес</string>
    </property>
   </widget>
   <widget class="QLabel" name="fullAddressLabel">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>570</y>
      <width>441</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>510</y>
      <width>20</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
